src/twisted/conch/test/test_keys.py

```diff
@@ -1139,6 +1147,62 @@ def test_blobEd25519(self):
             common.NS(b"ssh-ed25519") + common.NS(publicBytes),
         )
 
+    @skipWithoutEd25519
+    def test_blobSKEd25519(self):
+        """
+        Tests that the correct blob is returned after parse for a sk-ssh-ed25519@openssh.com public key.
```

Comment:
Minor comment... what is "correct blob" ?
 
Asked in the context of this article https://jml.io/test-docstrings/